### DISCUSSION TITLE
How and where to edit Amavis settings

### DISCUSSION THREAD

**Discussion by xword9009** [modoboa/modoboa#3044] [Help]
Hi,

I've recently installed v 2.1.2 and some of my users are getting quite a few outbound messages quarantined for failing bad header checks. 

While I can see a way to update the sql such that inbound emails will get through, but I’m unsure if setting the users as bad_header_lover effects outbound as well as inbound.

This seems relevant  [https://github.com/modoboa/modoboa-amavis/issues/112](https://github.com/modoboa/modoboa-amavis/issues/112) but I cannot find where to put the suggested code, there is no amavisd.conf.

Aplogies in andvanced if this is an especially dumb question, this is the 1st time I've run Modoboa.

Kind regards

Nigel

**Comment by H-A-L-I-S** [comment]
Did you try to search for amavis files on your system? Also try asking your favorite search engine for amavis config file
